openmetadata-ui/src/main/resources/ui/src/components/DataInsightDetail/DataInsightSummary.tsx

```diff
@@ -169,7 +181,10 @@ const DataInsightSummary: FC<Props> = ({ chartFilter, onScrollToChart }) => {
       loading={isLoading}
       title={
         <Typography.Title level={5}>
-          {t('label.data-insight-summary')}
+          {t('label.data-insight-summary', {
+            organization:
+              OrganizationDetails.displayName || t('label.openMetadata'),
```

Comment:
@Ashish8689, We should use `getEntityName` util, so if displayName is not present it fallback to name.